plugins/inputs/mysql/mysql.go

```diff
@@ -24,6 +30,10 @@ var sampleConfig = `
   #
   # If no servers are specified, then localhost is used as the host.
   servers = ["tcp(127.0.0.1:3306)/"]
+  PerfEventsStatementsDigestTextLimit = 120
```

Comment:
Can you add comments for all of these? and also change them to match the general underscore style of the config file,

ie:

```
perf_events_statement_digest_text_limit = 120
```